Vagrantfile

```diff
@@ -0,0 +1,115 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  config.vm.box = "boxcutter/ubuntu1604"
+
+
+    config.vm.network "forwarded_port", guest: 3000, host: 3000, auto_correct: true
+    config.vm.network "forwarded_port", guest: 80, host: 8080, auto_correct: true
```

Comment:
I use this vagrant file for development  and sometimes I setup apache/passenger to test things out. so that's why all the stuff is in there.

My question is, do you want vagrantfile setup for development or as a demo machine with apache/passenger setup?